src/Modules/Reports/Commands.Reports.Test/GetPowerBIReportTests.cs

```diff
@@ -15,34 +14,41 @@ namespace Commands.Reports.Test
     [TestClass]
     public class GetPowerBIReportTests
     {
+        private static CmdletInfo Cmdlet => new CmdletInfo($"{GetPowerBIReport.CmdletVerb}-{GetPowerBIReport.CmdletName}", typeof(GetPowerBIReport));
+
         [TestMethod]
         [TestCategory("Interactive")]
         [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
-        public void EndToEndGetReports()
+        public void EndToEndGetReportsIndividualScope()
         {
-            using (var ps = System.Management.Automation.PowerShell.Create())
+            using (var ps = PowerShell.Create())
             {
-                ps.AddCommand(new CmdletInfo($"{ConnectPowerBIServiceAccount.CmdletVerb}-{ConnectPowerBIServiceAccount.CmdletName}", typeof(ConnectPowerBIServiceAccount)));
-                var result = ps.Invoke();
-                Assert.IsNotNull(result);
-                ps.Commands.Clear();
-                ps.AddCommand(new CmdletInfo($"{GetPowerBIReport.CmdletVerb}-{GetPowerBIReport.CmdletName}", typeof(GetPowerBIReport)));
-                result = ps.Invoke();
-                Assert.IsNotNull(result);
-                Assert.IsTrue(result.Count > 0);
+                ProfileTestUtilities.ConnectToPowerBI(ps);
+
+                ps.AddCommand(Cmdlet);
+
+                var results = ps.Invoke();
+
+                Assert.IsNotNull(results);
+                if (!results.Any())
+                {
+                    Assert.Inconclusive("No reports returned. Verify you are assigned or own any reports.");
+                }
```

Comment:
Check for errors too

Reply:
I went to each test class and added the cmdlet error checking where it was missing.